2.2/Dockerfile

```diff
@@ -21,14 +21,13 @@ RUN set -x \
 	&& chmod +x /usr/local/bin/gosu \
 	&& gosu nobody true
 
-# https://www.elastic.co/guide/en/logstash/2.0/package-repositories.html
-# https://packages.elasticsearch.org/GPG-KEY-elasticsearch
-RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys 46095ACC8548582C1A2699A9D27D666CD88E42B4
+# https://www.elastic.co/guide/en/logstash/2.2/package-repositories.html
+RUN wget -qO - https://packages.elastic.co/GPG-KEY-elasticsearch | apt-key add -
```

Comment:
Agreed, I will change this but update the link to the documentation to the corresponding versions.